tests/unit/tasks/install-blueprint-test.js

```diff
@@ -1,18 +1,18 @@
 'use strict';
 
-const RSVP = require('rsvp');
 const path = require('path');
 const fs = require('fs-extra');
 const td = require('testdouble');
 const SilentError = require('silent-error');
 
 const expect = require('../../chai').expect;
+const promisify = require('util').promisify;
 const mkTmpDirIn = require('../../../lib/utilities/mk-tmp-dir-in');
 const InstallBlueprintTask = require('../../../lib/tasks/install-blueprint');
 
 let root = path.join(__dirname, '../../..');
 let tmproot = path.join(root, 'tmp');
-const remove = RSVP.denodeify(fs.remove);
+const remove = promisify(fs.remove);
```

Comment:
I think this can be removed completely. `fs.remove()` (at least from `fs-extra`) will always return a Promise by default now.